rke/k8s_rancher_versions.go

```diff
@@ -0,0 +1,89 @@
+package rke
+
+import "github.com/rancher/types/apis/management.cattle.io/v3"
+
+func loadRancherDefaultK8sVersions() map[string]string {
+	return map[string]string{
+		"2.3":     "v1.14.1-rancher1-2",
+		"default": "v1.14.1-rancher1-2",
```

Comment:
@kinarashah can you describe how 2.3 and default keys are gonna be used?

Reply:
I added it because currently don't have base rancher version to start with.. ideally it shouldn't fall back on "default" in actual setups. Should we let it be "default" or make it "dev"?

Reply:
default is fine. Can we just add a comment describing the ^?